tests/java/som/interpreter/objectstorage/SafepointPhaserTest.java

```diff
@@ -0,0 +1,96 @@
+package som.interpreter.objectstorage;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.concurrent.BrokenBarrierException;
+import java.util.concurrent.CyclicBarrier;
+
+import org.graalvm.collections.EconomicSet;
+import org.junit.Test;
+
+import som.compiler.AccessModifier;
+import som.compiler.MixinDefinition.SlotDefinition;
+import som.tests.ParallelHelper;
+import som.vmobjects.SClass;
+import som.vmobjects.SObject.SMutableObject;
+
+
+public class SafepointPhaserTest {
+
+  private final SClass       instanceClass;
+  private final ClassFactory factory;
+  private final ObjectLayout layout;
+
+  public SafepointPhaserTest() {
+    SlotDefinition slotDef = new SlotDefinition(null, AccessModifier.PUBLIC, false, null);
+    EconomicSet<SlotDefinition> instanceSlots = EconomicSet.create();
+    instanceSlots.add(slotDef);
+
+    instanceClass = new SClass(null);
+    factory = new ClassFactory(null, null, instanceSlots, null, false, false, false, null,
```

Comment:
This seems to have a wrong number of arguments on my machine.

Reply:
Hm, what exactly do you mean with this?
Also, is this without any other changes? Seems to compile fine on Travis and Brussels.